test/email-field-events.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>vaadin-email-field events tests</title>
+
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../vaadin-email-field.html">
+  <link rel="import" href="helpers.html">
+  <link rel="import" href="../../test-fixture/test-fixture.html">
+</head>
+
+<body>
+  <test-fixture id="email-field">
+    <template>
+      <vaadin-email-field></vaadin-email-field>
+    </template>
+  </test-fixture>
+<script>
```

Comment:
Please, align `<script>` with `<test-fixture>`. I thought about using a formatter afterward but it is easier if we format it correctly already now.